tests/runtime/resource_borrow_simple/wasm.rs

```diff
@@ -0,0 +1,17 @@
+wit_bindgen::generate!({
+    path: "../../tests/runtime/resource_borrow_simple",
+    exports: {
+        world: Test,
+    }
+});
+
+pub struct Test {}
+
+impl Guest for Test {
+    fn test_imports() {
+        unsafe {
+            let r = R { handle: wit_bindgen::rt::Resource::from_handle(0) };
```

Comment:
This doesn't feel good to use... If a resource doesn't have a way to construct a owned handle, how is the guest suppose to do? @alexcrichton

Reply:
Nevermind, this isn't working

Reply:
The WIT will need to have a `constructor()` or some other means by which the guest can create a resource, because otherwise there's no valid way for the guest to get a resource.